sky/serve/serve_utils.py

```diff
@@ -580,6 +575,11 @@ def _get_replica_status() -> serve_state.ReplicaStatus:
           f'of replica {replica_id}...{colorama.Style.RESET_ALL}')
 
     backend = backends.CloudVmRayBackend()
+    handle = global_user_state.get_handle_from_cluster_name(
```

Comment:
Move `handle = global_user_state.get_handle_from_cluster_name(...)` before `# Notify user here to make sure user won't think the log is finished.`?